templates/participants.html

```diff
@@ -68,3 +79,63 @@ <h1>Wszyscy uczestnicy</h1>
     </article>
 </div>
 {% endblock %}
+
+{% block script %}
+  <script>
+    $(document).ready(() => {
+      const table = $('#participants-table').DataTable({
```

Comment:
The search option searches in everything, including e.g. qualification points percentage numbers - I think it should be limited to only a few columns (name, email + maybe school and comments)